build.yaml

```diff
@@ -495,7 +496,7 @@ common_cli:
     # we provide one by starting a webserver and pointing the tests
     # to look over there
     REPO_PATH=/results/omnibus/${TRIGGERED_BRANCH}/${DISTRIBUTION_NAME}
-    DOC_ROOT=/usr/share/nginx/html
+    DOC_ROOT=/var/www/html
```

Comment:
This seems to be the standard directory for Nginx on Ubuntu 16.04 and all tests that require a local package repository are now run from Ubuntu 16.04.